src/errors/GitFileSystemNeedRollbackError.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseIsomerError } from "./BaseError"
```

Comment:
Reg PR description
>Pushes will happen synchronously so that the user can continue working on the CMS while the backend pushes to GitHub.

You mean async right?

Reply:
nit: better to name branches with ticket nums, can copy paste from jira
![Screenshot 2023-08-10 at 1 13 19 PM](https://github.com/isomerpages/isomercms-backend/assets/5507822/bcb1f8c5-11c8-4a0d-894d-16891f41e725)

Reply:
Yup oops I meant async, adjusted the description! Also noted on the branch name, thanks!